src/main/java/org/spongepowered/api/data/key/Keys.java

```diff
@@ -1458,6 +1461,61 @@
      */
     public static final Key<Value<LogAxis>> LOG_AXIS = DummyObjectProvider.createExtendedFor(Key.class,"LOG_AXIS");
 
+    /**
+     * Represents the {@link Key} for the {@link MapCanvas} of a map
+     * This contains the colors displayed on a map
+     *
+     * @see MapItemData#canvas()
+     */
+    public static final Key<Value<MapCanvas>> MAP_CANVAS = DummyObjectProvider.createExtendedFor(Key.class, "MAP_CANVAS");
+    /**
+     * Represents the {@link Key} for the world for a
+     * {@link ItemTypes#FILLED_MAP}
+     *
+     * @see MapItemData#world()
+     */
+    public static final Key<Value<World>> MAP_WORLD = DummyObjectProvider.createExtendedFor(Key.class, "DIMENSION");
+
+    /**
+     * Represents the {@link Key} for the centre x and z of where a
+     * {@link ItemTypes#FILLED_MAP} represents
+     *
+     * @see MapItemData#location()
+     */
+    public static final Key<Value<Vector2i>> MAP_LOCATION = DummyObjectProvider.createExtendedFor(Key.class, "MAP_LOCATION");
+
+    /**
+     * Represents the {@link Key} for whether a map tracks player
+     *
+     * @see MapItemData#trackPosition()
+     */
+    public static final Key<Value<Boolean>> MAP_TRACKS_PLAYERS = DummyObjectProvider.createExtendedFor(Key.class, "MAP_TRACKS_PLAYERS");
+
+    /**
+     * Represents the {@link Key} for whether a map can track
+     * a player from anywhere in the world
+     *
+     * @see MapItemData#unlimitedTracking()
+     */
+    public static final Key<Value<Boolean>> MAP_UNLIMITED_TRACKING = DummyObjectProvider.createExtendedFor(Key.class, "MAP_UNLIMITED_TRACKING");
+
+    /**
+     * Represents the {@link Key} for the scale of a map
+     *
```

Comment:
Nitpick: Maybe can link to the Minecraft wiki what scale means or how it's interpreted?